spec/factories/furniture/marketplace.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :marketplace do
+    transient do
+      room { build(:room) }
+    end
+
+    placement do
+      association :furniture_placement, { furniture_kind: 'marketplace', room: room }
+    end
+  end
+
+  factory :marketplace_product, class: 'Marketplace::Product' do
+    name { Faker::TvShows::DrWho.specie }
```

Comment:
Nice Faker choice. 👽

Reply:
I wanted one that was more nerdy equipment but I can't always get what I want!